extensions/guacamole-auth-ldap/src/main/java/org/apache/guacamole/auth/ldap/AuthenticationProviderService.java

```diff
@@ -189,7 +197,7 @@ private LDAPConnection bindAs(Credentials credentials)
 
     /**
      * Returns an AuthenticatedUser representing the user authenticated by the
-     * given credentials.
+     * given credentials. Also adds custom LDAP attributes to credentials object.
```

Comment:
Might want to update this comment to reflect the change from storing in `Credentials` to `AuthenticatedUser`